tests/test_multi_result.py

```diff
@@ -0,0 +1,112 @@
+##########################################################################
+# Copyright (c) 2010-2022 Robert Bosch GmbH
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0.
+#
+# SPDX-License-Identifier: EPL-2.0
+##########################################################################
+
+import sys
+
+import pytest
+
+from pykiso.test_result.multi_result import MultiTestResult
+from pykiso.test_result.text_result import BannerTestResult
+from pykiso.test_result.xml_result import XmlTestResult
+
+
+@pytest.fixture
+def multi_result_instance_multiple_classes():
+    return MultiTestResult(BannerTestResult, XmlTestResult)(sys.stderr, True, 1)
+
+
+@pytest.fixture
+def test_mock(mocker):
+    return mocker.patch("pykiso.test_coordinator.test_case.BasicTest")
+
+
+@pytest.mark.parametrize(
+    "name_function,argument",
+    [
+        ("startTest", {test_mock}),
+        ("startTestRun", {}),
+        ("stopTestRun", {}),
+        ("stop", {}),
+        ("stopTest", {test_mock}),
+        ("addFailure", {test_mock, Exception}),
+        ("addSuccess", {test_mock}),
+        ("addSkip", {test_mock, "reason"}),
+        ("addUnexpectedSuccess", {test_mock}),
+        ("addExpectedFailure", {test_mock, Exception}),
+        ("addSubTest", {test_mock, "subtest", Exception}),
+        ("addError", {test_mock, Exception}),
+    ],
+)
+def test_call_function(
+    mocker, multi_result_instance_multiple_classes, name_function, argument
+):
+    mock_xmltestresult = mocker.patch.object(XmlTestResult, name_function)
+    mock_bannertestresult = mocker.patch.object(BannerTestResult, name_function)
+
+    getattr(multi_result_instance_multiple_classes, name_function)(*argument)
+
+    mock_xmltestresult.assert_called_once_with(*argument)
+    mock_bannertestresult.assert_called_once_with(*argument)
+
+
+@pytest.mark.parametrize("attribute_name", ["errors", "failures"])
+def test___getattr__(multi_result_instance_multiple_classes, attribute_name):
+
+    attr = getattr(multi_result_instance_multiple_classes, attribute_name)
+
+    assert attr == []
```

Comment:
You are testing your __getattr__ by testing that it return an empty variable, it's super weird but at least if it do the job

Reply:
Test have been modified.